specification/cognitiveservices/data-plane/Language/preview/2022-10-01-preview/analyzeconversations-authoring.json

```
@@ -2523,7 +2523,7 @@
       "type": "object",
       "properties": {
         "expressions": {
-          "description": "The regex expressions of the regex component.",
+          "description": "The regex expressions of the regex component. These expressions follow the .NET regex syntax. For reference, check here: https://learn.microsoft.com/en-us/dotnet/standard/base-types/regular-expressions",
```

Do not include the locale in URLs as this forces everyone to en-US instead of using their browser to decide:
```suggestion
          "description": "The regex expressions of the regex component. These expressions follow the .NET regex syntax. For reference, check here: https://learn.microsoft.com/dotnet/standard/base-types/regular-expressions",
```

Same for any URL below.